akka-http/src/main/scala/akka/http/scaladsl/server/Route.scala

```diff
@@ -22,6 +22,8 @@ object Route {
 
   /**
    * "Seals" a route by wrapping it with exception handling and rejection conversion.
+   *
+   * A "sealed" route accepts all requests making any routes after it unreachable at runtime.
```

Comment:
hm I find this being weird wording. It'd be better to explain that it converts rejections to responses etc